containers/daap-delete-data-product/CHANGELOG.md

```diff
@@ -9,6 +9,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.1.0] 2023-11-21
+
+### Added
+
+- `database_names` fixture to generate a list of database names
+    for all major versions
+
+### Changed
+
+- updated `create_glue_database` to `create_glue_databases`
+- `create_glue_databases` now creates databases for all major versions
```

Comment:
I think this is a little confusing as it's referring to test code. I think this changelog should focus on the observable behaviour of the lambda.

Reply:
Removed